src/transport/PointToPointBroker.cpp

```diff
@@ -502,16 +518,73 @@ std::set<int> PointToPointBroker::getIdxsRegisteredForGroup(int groupId)
     return groupIdIdxsMap[groupId];
 }
 
+void PointToPointBroker::initSequenceCounters(int groupId)
+{
+    if (currentGroupId != NO_CURRENT_GROUP_ID) {
+        SPDLOG_WARN("Changing the current group Id in PTP broker ({} -> {})",
+                    currentGroupId,
+                    groupId);
+    }
+    currentGroupId = groupId;
+    int groupSize = getIdxsRegisteredForGroup(groupId).size();
+    // We assume that group indexes will always go from 0 to `groupSize - 1`
```

Comment:
This will always be true, but writing "we assume" here, suggests that there may not be groups for which this is the case. I think it's clearer not to have this line at all.